test/integration/component/test_escalation_listTemplatesDomainAdmin1.py

```diff
@@ -0,0 +1,94 @@
+
+from marvin.cloudstackTestCase import cloudstackTestCase
+from marvin.lib.base import (Account,
+                             Domain, Template
+                             )
+from marvin.lib.utils import *
+from marvin.lib.common import (get_zone)
+
+
+class Domain12(cloudstackTestCase):
+
+    @classmethod
+    def setUpClass(cls):
+
+        testClient = super(Domain12, cls).getClsTestClient()
+        cls.apiclient = testClient.getApiClient()
+        cls.testdata = testClient.getParsedTestDataConfig()
+        cls.testdata["templateregister"]["ostype"] = "Windows 7 (64-bit)"
+        cls.testdata["templateregister"]["url"] = "http://10.147.28.7/templates/windows7.vhd"
```

Comment:
Avoid hard coding of data in test case itself. Is windows template necessary? If not use the same template that is already registered in the zone. You can use get_builtin_template_info function from lib/common.py file. This has been used in many existing test cases.